defaults-prod-latest.sh

```diff
@@ -0,0 +1,23 @@
+package: defaults-prod-latest
+version: v1
+env:
+  CXXFLAGS: "-fPIC -g -O2 -std=c++11"
+  CFLAGS: "-fPIC -g -O2"
```

Comment:
probably the `-g` should be removed right away

Reply:
Hi @matthiasrichter, since `prod` aims to be the new `release` for central Run 2 builds. Those flags are correct in that context. Please ignore them for O2.

Reply:
...and since you quote #820: what you say is not true, we have `CXXFLAGS` and `CFLAGS` because _all_ recipes need to work with them. The build type (`CMAKE_BUILD_TYPE`) only applies to CMake projects.

Reply:
Fine for `prod`, however `user` is exposed to users. The flags as they are right now keep on producing problems in that context. Adding new files seems to me the right moment to clean this up. Just my personal take. I'm not using AliRoot anyhow.

Reply:
`user` for now is, again, exposed to Run 2 users, but it hasn't been announced or advertised. The idea is, indeed, to have some convergency for local builds between Run 2 and Run 3. Since #820 to me is a workaround to the for-now non-existing ability to override build flags per package (remember our discussion about overriding `env`?), can you remind what are the problems being produced in case `-g` is used?